### PR TITLE
Delete scroll if form is small

### PR DESCRIPTION
**Pull request by shastakova-trulioo**
Eslint/prettier fixed indents that's why it seems to have a lot of changes but actual changes start from line 78

### CODE REVIEW

File: src/TruliooClient.js [hunk 1]
```diff
@@ -1,90 +1,91 @@
...
-        embedIDModule.style.left = '0';
-        embedIDModule.style.position = 'absolute';
-        embedIDModule.style.top = '0';
-        embedIDModule.style.width = '100%';
-    }
-}
\ No newline at end of file
+    // styling of the iframe
+    const embedIDModule = document.getElementById('embedid-module');
+    embedIDModule.style.border = '0';
```
Comment (mkotsollaris) [inline src/TruliooClient.js]: This was there before, but can you create a function i.e.: `addStyles` that applies all the extra CSS?

File: src/TruliooClient.js [hunk 2]
```diff
@@ -1,90 +1,91 @@
 export default class TruliooClient {
+  constructor(obj) {
+    for (var fld in obj) {
+      this[fld] = obj[fld];
+    }
+    this.embedIDURL = this.embedIDURL
+      ? this.embedIDURL
+      : 'http://localhost:8855/embedid';
```
Comment (mkotsollaris) [inline src/TruliooClient.js]: please change this to https://embedid.trulioo.com/

**Comment by mkotsollaris** [comment]
make sure you run `npm run webpack-release` to bundle `TruliooClient.js`

File: src/TruliooClient.js (hunk 2 above)
Comment (shastakova-trulioo) [reply src/TruliooClient.js]: do you mean https://embedid.trulioo.com/embedid ?

Comment (mkotsollaris) [reply src/TruliooClient.js]: yes*

File: src/TruliooClient.js (hunk 1 above)
Comment (shastakova-trulioo) [reply src/TruliooClient.js]: created, thanks

File: src/TruliooClient.js [hunk 3]
```diff
@@ -1,90 +1,94 @@
 export default class TruliooClient {
+  constructor(obj) {
+    for (var fld in obj) {
+      this[fld] = obj[fld];
+    }
+    this.embedIDURL = this.embedIDURL
+      ? this.embedIDURL
+      : 'https://embedid.trulioo.com/embedid';
```
Comment (shastakova-trulioo) [inline src/TruliooClient.js]: added production url since we have it now

File: src/TruliooClient.js [hunk 4]
```diff
@@ -1,90 +1,94 @@
...
-                // call function
-                if (truliooClient[data.function]) {
-                    truliooClient[data.function].call(window, JSON.parse(data.message));
-                } else {
-                    console.log(`Tried to trigger ${data.func} but found no available callBack`);
-                }
-            } catch (error) {
-                console.error('Something went wrong during callback registration', error);
-            }
+        // call function
```
Comment (mkotsollaris) [inline src/TruliooClient.js]: probably unnecessary comment?

File: src/TruliooClient.js [hunk 5]
```diff
@@ -1,90 +1,94 @@
...
+  }
 
-    loadEmbedID() {
-        const url = `${embedIDURL}/${this.publicKey}/at/${this.accessToken}`;
-        const element = document.createElement('iframe');
-        element.setAttribute('id', 'embedid-module');
-        element.setAttribute('src', url);
-        const truliooEmbedIDContainer = document.getElementById('trulioo-embedid');
-        truliooEmbedIDContainer.appendChild(element);
+  addBasicIframeStyles() {
```
Comment (mkotsollaris) [inline src/TruliooClient.js]: Please pass `document.getElementById('embedid-module');` from the calling function

File: src/TruliooClient.js (hunk 4 above)
Comment (shastakova-trulioo) [reply src/TruliooClient.js]: deleted

File: src/TruliooClient.js (hunk 5 above)
Comment (shastakova-trulioo) [reply src/TruliooClient.js]: done

**Pull request merged**